src/services/itemMembership/repository.ts

```diff
@@ -442,6 +443,16 @@ export const ItemMembershipRepository = AppDataSource.getRepository(ItemMembersh
     });
   },
 
+  async getAllWithPermission(
```

Comment:
What do you think about the following name ? 

```suggestion
  async getByItemPathAndPermission(
```